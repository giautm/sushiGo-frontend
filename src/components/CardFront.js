import React from 'react';

const CardFront = props => {
  return (
    <img
      alt={`sushi ${props.rollData.imageNumber}`}
      src={require(`../images/Sushi-${props.rollData.imageNumber}.png`)}
    />
  );
};

export default CardFront;
