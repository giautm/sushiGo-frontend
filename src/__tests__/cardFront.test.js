import React from 'react';
import CardFront from '../components/CardFront';
import renderer from 'react-test-renderer';

const fakeRollData = {
  id: '1',
  imageNumber: '8',
  name: 'Spicy Tuna Roll',
  ingredients: 'Tuna, Chili sauce, Nori, Rice'
};

describe('CardFront', () => {
  it('renders correctly', () => {
    const component = renderer.create(<CardFront rollData={fakeRollData} />);
    expect(component).toMatchSnapshot();
  });
});
