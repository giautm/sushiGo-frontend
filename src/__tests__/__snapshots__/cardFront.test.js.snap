// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFront renders correctly 1`] = `
<img
  alt="sushi 8"
  src={
    Object {
      "process": [Function],
    }
  }
/>
`;
